src/script/lua_api/l_util.cpp

```diff
@@ -423,6 +425,31 @@ int ModApiUtil::l_get_version(lua_State *L)
 	return 1;
 }
 
+int ModApiUtil::l_sha1(lua_State *L)
+{
+	NO_MAP_LOCK_REQUIRED;
+	std::string data = luaL_checkstring(L, 1);
+	bool hex = !lua_isboolean(L, 2) || !lua_toboolean(L, 2);
+
+	// Compute actual checksum of data
+	std::string data_sha1;
+	{
+		SHA1 data_sha1_calculator;
```

Comment:
what kind of variable name is this?
just call it `ctx`

Reply:
Good point. I had copied it from somewhere else in minetest where a sha1 was calculated.